activerecord/test/cases/associations/has_one_associations_test.rb

```diff
@@ -531,7 +531,7 @@ def test_attributes_are_being_set_when_initialized_from_has_one_association_with
     assert_equal "Account", new_account.firm_name
   end
 
-  def test_create_association_replaces_existing_without_dependent_option
+  def test_create_association_failure_replaces_existing_without_dependent_option
```

Comment:
Should it be "test_creation_failure_replaces...." instead? Similar to line 558 and how it was before?
```suggestion
  def test_creation_failure_replaces_existing_without_dependent_option
```

Reply:
Ya I guess association is already implied, I've updated 👍🏽